README.md

```diff
@@ -0,0 +1,87 @@
+# multiplex-reservation
+
+## Table of contents
+* [Informacje Ogólne](#informacje-ogólne)
+* [Komunikacja z aplikacją](#komunikacja-z-aplikacją)
+* [Warstwa Service](#warstwa-service)
+* [Generowanie Kina](#generowanie-kina)
+* [Odnośnie wymagań](#odnośnie-wymagań)
+
+## Informacje Ogólne
+
+Aplikacja webowa napisana w języku Java z użyciem frameworka Spring umozliwuająca rezerwację miejsc w kinie typu multiplex.
+Aplikacja nie posiada zaimplementowanego frontendu. Komunikowałem się z nią i testowałem za pomocą aplikacji Postman.
```

Comment:
Postman, hmmm, a czemu nie testy? :)

Reply:
Gdzieś w internecie wyczytałem, że jest to super łatwa do obsługi i przejrzysta aplikacja do tego typu testów. Szczerze nawet nie myślałem o implementacji kodu testowego.